docs/4.0/docs/cloud/apps/scp/scp.md

```diff
@@ -102,7 +102,7 @@ Download the user-specific kubeconfig file from the sealos cloud:
 
 ![img_4.png](img_4.png)
 
-Copy to the .kube/config file, e.g. `cp kubeconfig.yaml .kube/config`
+Copy it to the .kube/config folder, e.g. `cp kubeconfig.yaml .kube/config`
```

Comment:
`.kube/config` is the kubeconfig file, not the folder stores it. So just keep original and do not changed to `folder`.

Reply:
Modified.